wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -20,6 +18,7 @@
         {% if parent_page %}
             {% page_permissions parent_page as parent_page_perms %}
             <tr class="index {% if not parent_page.live %} unpublished{% endif %} {% block parent_page_row_classname %}{% endblock %}">
+                <td></td>
                 <td class="title" {% if show_ordering_column %}colspan="2"{% endif %}>
```

Comment:
```suggestion
                <td class="title" colspan="2">
```
would keep the page title consistent and in line with the left edge of the breadcrumb as well as the first column below